### ISSUE TITLE
CRA Failed to minify the code from this file: mobx.module.js:23

### ISSUE THREAD

**Issue by ashconnell** [mobxjs/mobx#1584]
This [related issue](https://github.com/mobxjs/mobx/issues/1379) was fixed but it looks like it might be back in 5.0.0
```
ashconnell$ yarn build
yarn run v1.7.0
$ react-scripts build
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

        ./node_modules/mobx/lib/mobx.module.js:23

Read more here: http://bit.ly/2tRViJ9

error Command failed with exit code 1.
```

Quickest way to reproduce is to `git clone` [this repo](https://github.com/ashconnell/mobx-quantum) and run `yarn && yarn build`

**Comment by mweststrate** [comment]
I have some idea what is happening here and will investigate further on monday, but first a double check, you are using the latest version of CRA?

**Comment by feat7** [comment]
I got into same issue today. After a lot of research and different attempts to solve the error, everything went in vain.
I am using `custom-react-scripts` with `create-react-app`
I managed to solve this issue by downgrading to mobx@4.3.1 (any version of 4 is working fine)
Hope this temporarily fix the issue.

**Comment by ashconnell** [comment]
@mweststrate i used `npx create-react-app ...` which i believe always uses the latest version of a package,

You can get a minimal reproduction with this:

```
npx create-react-app mobx-5-cra-build
cd mobx-5-cra-build
yarn add mobx
echo "import { observable } from 'mobx'" > src/index.js
yarn build
```

**Comment by mweststrate** [comment]
Thanks for the minimal reproduction @ashconnell.

I reverted MobX back to using ES5 only. That should solve thte issue. Released as `5.0.1`

**Comment by mweststrate** [comment]
Pro tip, the default build is now ES5, but you can still use the smaller and faster build by setting up an alias. Webpack example:  `resolve: { alias: { mobx: __dirname + "/node_modules/mobx/lib/mobx.es6.js" }}`
